locales/en/yourloops.json

```diff
@@ -83,6 +83,8 @@
   "current-password": "Current password",
   "date-picker-header-date-format": "MMMM YYYY",
   "date-picker-toolbar-date-format": "ddd, MMM D",
+  "diabeto": "Diabetologist",
```

Comment:
I would have prefix the keys with some like "hcp-job-" for example
So we have some context in crowdin for the translations.

Reply:
Done